File: contest/abc229/src/bin/b.py
(a, b) = input().split()
al = len(a)
bl = len(b)

l = min(al, bl)

flag = False
for i in range(l):
  ai = int(a[al-i-1])
  bi = int(b[bl-i-1])
  # print(ai, bi)
  if ai + bi > 9:
    flag = True
    break
  
  
if flag:
  print("Hard")
else:
  print("Easy")

File: contest/abc229/src/bin/b.rs
use proconio::input;
fn main() {
  input! {
    a: String,
    b: String
  }
  let al = a.len();
  let bl = b.len();
  let l = if al < bl { al } else {bl};

  for i in 0..l {
    let ai: usize = a[i].parse().unwrap();
    let bi: usize = b[i].clone().parse().unwrap();
  }

  ans(true);
}

fn ans(flag: bool) {
  if flag {
    println!("Easy")
  } else {
    println!("Hard")
  }
}
